### DISCUSSION TITLE
Link from next/link always hard-navigate on dynamic route if newly created in Vercel

### DISCUSSION THREAD

**Discussion by ShueiYang** [vercel/next.js#54613] [App Router]
Hello! 

I noticed yesterday for some unknown reason when I try for example to add a new product on a page having a Link component, it will 100% hard reload the entire page (like a html tag)  when I click on it, and unfortunaly even on the latest version 13.4.19 i get the same result, I remember that wasn't the case before so i downgraded to 13.4.9 but it change nothing, it happen ofc only if the app is deployed on Vercel, it work perfectly fine in local build, as well in Docker build host in other provider, so I wonder if there is people having this kind of issue ?

**Comment by estevaz12** [comment]
I'm experiencing the same issue

**Comment by matteo-naif** [comment]
I have the same problem: I need to handle in the same [Catch-all Segments](https://nextjs.org/docs/app/building-your-application/routing/dynamic-routes#catch-all-segments) the catalog and clicking a Link compnent to change the routing to apply a facet, it hard reload.

* Routing: `[...slug]`
* Current page: `domain/category-name/`
* Destination page: `domain/category-name/material--wood/`
